lower.ts

```diff
@@ -192,6 +193,38 @@ function flattenStmt(s : AST.Stmt<Type>, blocks: Array<IR.BasicBlock<Type>>, env
 
       blocks.push({  a: s.a, label: whileEndLbl, stmts: [] })
 
+      return [...cinits, ...bodyinits]
+    case "continue":
+      pushStmtsToLastBlock(blocks, { tag: "jmp", lbl:  env.labels[0]})
```

Comment:
Will these work within while loops as well?

If so, awesome, can we add a test or two demonstrating that?

Reply:
I did not test the continue/break statements in while loops. Will add those tests and necessary changes if any. Thanks for the suggestion.

Reply:
Week-8 implementation has required modifications an testcases for the break and continue statements to work in the while. loop and nested loops. Also, test cases to iterate over non-int iterators are included.  Thanks for the suggestion.